### DISCUSSION TITLE
Can not download videos for a website which the downloadhelper used to work a few days ago

### DISCUSSION THREAD

**Discussion by kamiofuju** [aclap-dev/video-downloadhelper#1359] [Help and Q&A]
### Your question

I cannot download the videos on the website (https://kamiofuju.com/videos) now. 
Video downloadhelper used to work a few days ago on this website but it fails to download the videos now.
Could you please help to confirm the reason and any sollutions for this problem?

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: zh-CN
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"79d702df0ea4444a"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\dell\\Desktop"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 11,
    "successfull_dl": 2,
    "record_download_history": true
}

**Comment by Mattmjs** [comment]
It is for members only plus the site made for Chrome & Edge. If your getting the failure message then click the report button. 
The report will be sent to the dev to investigate.
